src/control/cmd/daos_agent/main.go

```diff
@@ -251,6 +193,75 @@ func parseOpts(args []string, opts *cliOptions, invoker control.Invoker, log *lo
 	return err
 }
 
+func processConfig(log *logging.LeveledLogger, cmd flags.Commander, opts *cliOptions, cfgPath string) (*Config, error) {
```

Comment:
I think you probably want to use a `logging.Logger` here instead of the concrete type.

Reply:
Good catch.